src/Data/Algorand/Transaction.hs

```diff
@@ -118,29 +113,60 @@ data TransactionType
   deriving (Eq, Generic, Show)
 
 -- | Constants for @OnComplete@.
---
-type OnComplete = Text
-
-onCompleteNoOp :: OnComplete
-onCompleteNoOp = "call"
```

Comment:
hm.. interesting, where did you found `"call"`?
I see you do not use it anymore, but from what source did it came?

Reply:
The string I got is from here: https://github.com/algorand/py-algorand-sdk/blob/develop/test/steps/v2_steps.py#L1639-L1643. This seems to be for something else.

Reply:
thanks